build.gradle

```diff
@@ -127,6 +127,7 @@ apply plugin: 'org.jetbrains.intellij'
 intellij {
     pluginName = 'intellij-dlanguage'
     version = ideaVersion
+    instrumentCode = false
```

Comment:
is it possible to do the update without setting this to false?

Reply:
Yes but in this case we should delete the instrumented-JAR before the verifyPlugin task